beacon-chain/state/interfaces.go

```diff
@@ -227,6 +227,8 @@ type WriteOnlyCheckpoint interface {
 type WriteOnlyAttestations interface {
 	AppendCurrentEpochAttestations(val *ethpb.PendingAttestation) error
 	AppendPreviousEpochAttestations(val *ethpb.PendingAttestation) error
+	SetPreviousEpochAttestations([]*ethpb.PendingAttestation) error
+	SetCurrentEpochAttestations([]*ethpb.PendingAttestation) error
```

Comment:
I added these setters so that the `.empty` method of `premineGenesisConfig` could purely use the `state.BeaconState` interface to prepare the BeaconState.